src/App.vue

```diff
@@ -105,7 +105,7 @@
                 :offset="{ top: 86, bottom: 12 }"
               >
                 <!-- Corrections still use the basic Fee Summary component -->
-                <aside v-if="isCorrectionFiling">
+                <aside v-if="isCorrectionFiling && getFilingData.filingTypeCode && getFilingData.entityType">
```

Comment:
This is to pre-empt an error where the SbcFeeSummary calls the pay API to fetch the fees, but the data is initially incomplete, and this error is displayed:

![image](https://user-images.githubusercontent.com/10002889/177859171-061253f4-db8d-483d-9db1-4df3167c8cd0.png)

Now the SbcFeeSummary displays only once the data is complete.